### PR TITLE
Adding Metromap (with Sections)

### PR DESCRIPTION
**Pull request by Shaun-Regenbaum**
As pertaining to #383 and #395 and closing #437 and #436

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,8 @@
 
 **nf-core/methylseq** is a bioinformatics analysis pipeline used for Methylation (Bisulfite) sequencing data. It pre-processes raw data from FastQ inputs, aligns the reads and performs extensive quality-control on the results.
 
+![image](https://github.com/user-attachments/assets/9bec0af6-1eda-4d1d-9452-9dfba136ba6e)
```
Comment (maxulysse) [inline README.md:24]: Fix the link

**Review by maxulysse — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sateeshperi) [reply README.md:24]: I thought it was autogenerated by github ?

Comment (maxulysse) [reply README.md:24]: Probably from a drag and drop, but you want to point to the actual file

Comment (Shaun-Regenbaum) [reply README.md:24]: Fixing now one sec

**Review by Shaun-Regenbaum — COMMENTED** [review]
should be fixed?

File: README.md [hunk 2]
```diff
@@ -21,6 +21,8 @@
 
 **nf-core/methylseq** is a bioinformatics analysis pipeline used for Methylation (Bisulfite) sequencing data. It pre-processes raw data from FastQ inputs, aligns the reads and performs extensive quality-control on the results.
 
+![image](https://github.com/Shaun-Regenbaum/methylseq/blob/dev/docs/images/metromap.png)
```
Comment (sateeshperi) [inline README.md:24]:
```suggestion
![nf-core/methylseq metro map](docs/images/metromap.png)
```

**Review by Shaun-Regenbaum — COMMENTED** [review]
this should be the right change finally 🤣

File: README.md (hunk 2 above)
Comment (Shaun-Regenbaum) [reply README.md]: We made the same change 🤣

**Review by maxulysse — APPROVED** [review]

**Pull request merged**